controller_manager:
  ros__parameters:
    update_rate: 100  # Hz
    use_sim_time: true

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    rover_controller:
      type: joint_trajectory_controller/JointTrajectoryController

  

rover_controller:
  ros__parameters:
    joints:
      - steer1
      - steer2
      - steer_3
      - steer4
      - wheel_1
      - wheel_2
      - wheel_3
      - wheel_4
     
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity

  